content/enterprise_influxdb/v1.9/administration/manage/security/_index.md

```diff
@@ -1,14 +1,18 @@
 ---
-title: Manage security in InfluxDB Enterprise
-description: Protect the data in your InfluxDB Enterprise instance.
+title: Manage security
+description: Configuration, security, and logging in InfluxDB enterprise.
 menu:
   enterprise_influxdb_1_9:
     name: Manage security
-    weight: 110
-    parent: Administration
+    weight: 40
+    parent: Manage
+aliases:
+  - /enterprise_influxdb/v1.9/administration/security/
 ---
 
-Some customers may choose to install InfluxDB Enterprise with public internet access, however doing so can inadvertently expose your data and invite unwelcome attacks on your database.
+<!--
```

Comment:
Do we recommend they set all up? Curious does anything have to be done in a particular order? Like could someone set up and manage users and privileges with meta api before managing authentication and authorization